react/modules/OrderQueue.tsx

```diff
@@ -0,0 +1,9 @@
+import {
```

Comment:
why create this module that "only" re-exports from `@vtex/order-manager` instead of importing it directly in `react/OrderQueue.tsx`?